applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -13,6 +13,8 @@ rsource "bt_management/Kconfig"
 #----------------------------------------------------------------------------#
 menu "Bluetooth audio"
 
+rsource "bt_stream/broadcast/Kconfig"
```

Comment:
Is it possible to rsource this only if AUDIO_DEV = 2 && TRANSPORT_BIS?
Meaning moving this check to here

Reply:
The Kconfig doesn't need to be guarded, but we can add `visible if ...`